spec/requests/api/v1/items/create_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'item creation' do
+  it 'accepts json body and post to create' do
+    merchant = create(:merchant)
+    json_body = { "name": "new item", "description": "this is a new item", "unit_price": 1, "merchant_id": merchant.id}
+
+    post api_v1_items_path, params: json_body
+
+    expect(response.status).to eq(201)
+
+    reply = JSON.parse(response.body, symbolize_names: true)
+
+    expect(reply).to have_key(:data)
+    expect(reply[:data]).to have_key(:id)
+    expect(reply[:data]).to have_key(:type)
+    expect(reply[:data]).to have_key(:attributes)
+    expect(reply[:data][:attributes]).to have_key(:name)
+    expect(reply[:data][:attributes]).to have_key(:description)
+    expect(reply[:data][:attributes]).to have_key(:unit_price)
```

Comment:
For these expectations, would they still pass if the controller returned this? And if so, is that the behavior you want?

```ruby
reply = {
  data: {
    id: "",
    type: "BLAH!",
    attributes: {
      name: nil,
      description: 999999,
      unit_price: []
    }
  }
}
```

Reply:
good point